### ISSUE TITLE
Add a new organization to ROR: 
 
Digitisation Centre of Western Australia

### ISSUE THREAD

**Issue by mariagould** [ror-community/ror-updates#211]
Summary of request: Add a new organization to ROR

Name of organization: Digitisation Centre of Western Australia
Website: https://www.uwa.edu.au/schools/research/The-Digitisation-Centre-of-Western-Australia
Link to publications: https://www.uwa.edu.au/schools/research/The-Digitisation-Centre-of-Western-Australia
Organization type: 
Wikipedia page: 
Wikidata ID: 
ISNI ID: 
GRID ID: 
Crossref Funder ID: 
Other names for the organization: Digitisation Centre WA
Aliases: Digitisation Centre WA
Acronym/abbreviation: DCWA, WADC
Related organizations: Curtin University https://ror.org/02n415q13 (related), ECU https://ror.org/05jhnwe22  (related), Murdoch University https://ror.org/00r4sry34  (related), UNDA https://ror.org/02stey378  (related), UWA https://ror.org/047272k79 (related)
City: Perth
Country: Australia
Geonames ID: 2063523
Other information about this request: This is a new research entity in WA. A facility for digitisation of HASS Collections to be made available online for researchers.

**Comment by arthurpsmith** [comment]
Definitely a research facility. While mainly located at UWA it has staff and some equipment elsewhere, so it's not really solely a child organization of that university. Since it's very new we might want to wait on this one a little? But otherwise I think it should have a ROR id.

**Comment by nlundvick** [comment]
I concur with Arthur, this organization should be added to ROR. Although new, it is associated with established universities and funded by the Australian Research Council. I think it would be ok to add now but moving to needs discussion to determine if request should be put on hold.

**Comment by adambuttrick** [comment]
Spoke with ARDC on our community call and they need this added in order to begin research and funding acknowledgements. Falls within the scope of previous curation discussion about trusted groups providing verified requests for new organizations.

**Comment by adambuttrick** [comment]
Assigned ROR ID https://ror.org/00jjz9s23 in release v1.0.
